microsetta_public_api/api/tests/test_integration.py

```diff
@@ -18,6 +19,10 @@
 from microsetta_public_api.resources_alt import resources_alt, Q2Visitor
 
 
+logger = logging.getLogger()
+logger.setLevel(logging.WARNING)
```

Comment:
Is this to suppress the timing warnings?

Reply:
If so, it would be more appropriately scoped to do
```
from microsetta_public_api.logging import logger
logger.setLevel(logging.WARNING)
```

Reply:
Yes and can do